card/mod/01_core/spec/set/all/tracked_attributes_spec.rb

```diff
@@ -228,88 +233,88 @@ def cards_should_be_added number
+      it 'test_reference_updates_plus_to_simple' do
          c1 = Card::Auth.as_bot do
-           Card.create! name: 'Huck', content: "[[chuck wagn+chuck]]"
+           Card.create! name: 'Huck', content: '[[chuck wagn+chuck]]'
          end
-         c2 = Card["chuck_wagn+chuck"]
+         c2 = Card['chuck_wagn+chuck']
          assert_rename c2, 'schmuck'
          c1 = Card.find(c1.id)
          assert_equal '[[schmuck]]', c1.content
       end
     end
 
-    context "dairy" do
+    context 'dairy' do
       before do
         Card::Auth.as_bot do
-          Card.create! type: "Cardtype", name: "Dairy", content: "[[/new/{{_self|name}}|new]]"
+          Card.create! type: 'Cardtype', name: 'Dairy', content: '[[/new/{{_self|name}}|new]]'
         end
       end
 
-      it "test_renaming_card_with_self_link_should_not_hang" do
-        c = Card["Dairy"]
-        c.name = "Buttah"
+      it 'test_renaming_card_with_self_link_should_not_hang' do
+        c = Card['Dairy']
+        c.name = 'Buttah'
         c.update_referencers = true
         c.save!
-        assert_equal "[[/new/{{_self|name}}|new]]", Card["Buttah"].content
+        assert_equal '[[/new/{{_self|name}}|new]]', Card['Buttah'].content
       end
 
-      it "test_renaming_card_without_updating_references_should_not_have_errors" do
-        c = Card["Dairy"]
-        c.update_attributes "name"=>"Newt", "update_referencers"=>'false'
-        assert_equal "[[/new/{{_self|name}}|new]]", Card["Newt"].content
+      it 'test_renaming_card_without_updating_references_should_not_have_errors' do
+        c = Card['Dairy']
+        c.update_attributes 'name'=>'Newt', 'update_referencers'=>'false'
+        assert_equal '[[/new/{{_self|name}}|new]]', Card['Newt'].content
       end
     end
 
 
-    context "blues" do
+    context 'blues' do
       before do
         Card::Auth.as_bot do
-          Card.create! name: "Blue"
+          Card.create! name: 'Blue'
 
-          Card.create! name: "blue includer 1", content: "{{Blue}}"
-          Card.create! name: "blue includer 2", content: "{{blue|closed;other:stuff}}"
+          Card.create! name: 'blue includer 1', content: '{{Blue}}'
+          Card.create! name: 'blue includer 2', content: '{{blue|closed;other:stuff}}'
 
-          Card.create! name: "blue linker 1", content: "[[Blue]]"
-          Card.create! name: "blue linker 2", content: "[[blue]]"
+          Card.create! name: 'blue linker 1', content: '[[Blue]]'
+          Card.create! name: 'blue linker 2', content: '[[blue]]'
         end
       end
 
 
-      it "test_updates_inclusions_when_renaming" do
-        c1,c2,c3 = Card["Blue"], Card["blue includer 1"], Card["blue includer 2"]
-        c1.update_attributes name: "Red", update_referencers: true
-        assert_equal "{{Red}}", Card.find(c2.id).content
+      it 'test_updates_inclusions_when_renaming' do
+        c1,c2,c3 = Card['Blue'], Card['blue includer 1'], Card['blue includer 2']
```

Comment:
Line is too long. [81/80]<br>Do not use parallel assignment.<br>Space missing after comma.